dfdaemon/config/config.go

```diff
@@ -304,3 +382,26 @@ func NewProxy(regx string, useHTTPS bool, direct bool) (*Proxy, error) {
 func (r *Proxy) Match(url string) bool {
 	return r.Regx != nil && r.Regx.MatchString(url)
 }
+
+// Error is an error with exit code
+type Error struct {
```

Comment:
Use common.errors?

Reply:
Sure, didn't notice that we have it in the common package